components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -540,7 +540,23 @@ public enum ErrorMessages {
                 "Server encountered an error when deleting shared application roles related to" +
                         " parent application: %s in organization: %s."),
         ERROR_CODE_ERROR_DELETING_USER_ROLE_ASSIGNMENTS("65098", "Unable to remove the user from the roles.",
-                "Server encountered an error while removing the user id: %s from the roles.");
+                "Server encountered an error while removing the user id: %s from the roles."),
+        ERROR_CODE_ERROR_AUTHENTICATING_TENANT_DELETION_USER("65099", "Error while authenticating" +
+                " tenant deletion user",
+                "Error while sending request for generating authentication token for tenant deletion user"),
+        ERROR_CODE_ERROR_BUILDING_TENANT_DELETION_AUTH_REQUEST_BODY("65100", "Error while authenticating" +
+                " tenant deletion user",
+                "Error while sending request for generating authentication token for tenant deletion user"),
```

Comment:
Both has the same message and description.

Reply:
fixed with commit